spec/controllers/lists_controller_spec.rb

```diff
@@ -0,0 +1,51 @@
+require "rails_helper"
+
+describe ListsController do
+
+  before {sign_in}
+
+  describe "GET new" do
+    let(:board) {create(:board)}
+    it "renders the new template" do
+      get :new, board_id: board.id
+      expect(response).to render_template(:new)
+    end
+  end
+
+  describe "GET edit" do
+    let(:board) {create(:board)}
+    let(:list) {create(:list, board: board)}
+    it "renders the edit template" do
+      get :edit, board_id: board.id, id: list.id
+      expect(response).to render_template(:edit)
+    end
+  end
+
+  describe "DELETE destroy" do
+    let!(:board) {create(:board)}
```

Comment:
jak wyniesiesz leta wyzej to poprostu go wywołaj w `before { board }` i to zadziala jakbys zrobił let'a z wykrzyknikiem ;) Pamietaj zeby pisac DRY - do not repeat yourself